config/main.py

```diff
@@ -2239,6 +2208,11 @@ def breakout(ctx, interface_name, mode, verbose, force_remove_dependencies, load
         del_intf_dict.pop(item)
         add_intf_dict.pop(item)
 
+    # validate all del_ports before calling breakOutPort
+    for intf in del_intf_dict.keys():
+        if not interface_name_is_valid(intf):
+            raise Exception("Interface name {} is invalid")
```

Comment:
>Exception [](start = 18, length = 9)

why some raise exception, some raise abort?

Reply:
yes, it should be 'abort' in main.py not exception, let me fix it.

Reply:
Fixed